src/drivers/driver_zephyr.c

```diff
@@ -670,10 +670,6 @@ static int wpa_drv_zep_get_ssid(void *priv,
 
 	if_ctx = priv;
 
-	if (!if_ctx->ssid) {
-		return 0;
-	}
-
```

Comment:
Won't the further code crash if we remove this check?

Reply:
No, that is the warning, ssid is an array on the stack, it can never be NULL.